assignments/closure.js

```diff
@@ -1,11 +1,29 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+//var name = "Shannon"
+//const greeting = (name) => {
+//  var greetingWord = "Hello"
+//  return function () {return greetingWord + " " + name;}
+//}
+
+//console.log(greeting());
+var greeting = (function () {
```

Comment:
I feel like we did this portion together in terms of `let` but am unsure of whether or not this has been updated since then.